app/views/documents/ssa1099s/edit.html.erb

```diff
@@ -1,3 +1,9 @@
 <% content_for :form_question, "Attach your SSA-1099's" %>
 
-<% content_for :form_help_text, "Based on your answers, you should have received the following document." %>
+<% content_for :form_help_text do %>
+  <strong>You should have this document</strong>
+  because you said that you had income from Social Security.
```

Comment:
we should update this to be consistent with the railroad retirement benefits one - the original question was "did you have income from social security OR railroad etc" so we can't know that they had income from social security